compiler/core/lam_compile_external_call.ml

```diff
@@ -267,9 +267,9 @@ let translate_scoped_access scopes obj =
   | [] -> obj
   | x :: xs -> Ext_list.fold_left xs (E.dot obj x) E.dot
 
-let translate_ffi (cxt : Lam_compile_context.t) arg_types
-    (ffi : External_ffi_types.external_spec) (args : J.expression list)
-    ~dynamic_import =
+let translate_ffi ?(transformed_jsx = false) (cxt : Lam_compile_context.t)
```

Comment:
Do we get `transformed_jsx` true sometimes when compiling an external call?
Can't remember if the jsx ppx produced external calls.